mypy/nodes.py

```diff
@@ -390,6 +405,10 @@ def set_line(self, target: Union[Token, Node, int]) -> Node:
             self.initialization_statement.set_line(self.line)
             self.initialization_statement.lvalues[0].set_line(self.line)
 
+    def set_column(self, target: Union[Token, Node, int]) -> Node:
+        super().set_column(target)
+        return self
```

Comment:
and yep, will combine them/see how bad the fallout is if we take out the return self (think it shouldn't be that bad at all, from the places I've seen `set_line` so far)